EIPS/eip-5114.md

```diff
@@ -0,0 +1,75 @@
+---
+eip: 5114
+title: Soulbound Token
+description: A token that is attached to a "soul" at mint time and cannot be transferred after that.
+author: Micah Zoltu (@MicahZoltu)
+discussions-to: https://ethereum-magicians.org/t/eip-5114-soulbound-token/9417
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-05-30
+requires: 721
+---
+
+## Abstract
+A soulbound token is a token that is bound to another Non-Fungible Token (NFT; e.g., a EIP-721 token) when it is minted, and cannot be transferred/moved after that.
+
```

Comment:
I'm not sure why this EIP needs to exist. Are there many use cases for nontransferable tokens? Why should implementers prefer this EIP over, say, [EIP-4973](https://eips.ethereum.org/EIPS/eip-4973)?

Seems like this EIP needs a Motivation section ;)

Reply:
I see what you did there.  I may consider adding a motivation after this has merged as a draft, but as it isn't required I would like to move forward without it for now.